sdk-actors/src/main/java/io/dapr/actors/client/ActorProxyImpl.java

```diff
@@ -21,6 +21,8 @@
  */
 class ActorProxyImpl implements ActorProxy, InvocationHandler {
 
+  private static final String UNDEFINED_CLASS_NAME = "io.dapr.actors.Undefined";
```

Comment:
We cannot refer to the class since it is internal to the package in which it was declared.